src/components/styles/Layout.tsx

```diff
@@ -21,13 +21,14 @@ export const PageLayout = styled(Layout)`
   ${whiteBackground}
 `;
 
-const siderWidth = "275px";
-
 export const PageSider = styled(Sider)`
   ${whiteBackground}
-  width: ${siderWidth} !important;
-  max-width: ${siderWidth} !important;
-  min-width: ${siderWidth} !important;
+  width: ${(props: { width?: number }): string =>
+    props.width ? `${props.width}px` : "275px"}; !important;
```

Comment:
I think we can remove this width css styling according to the antd docs they provide a prop to update the width. https://ant.design/components/layout/#Layout.Sider 

I think it would be better to just  use the width prop instead of hacking the css. 

```
export const PageSider = styled(Sider)`
  ${whiteBackground}
`;
PageSider.defaultProps = { width: 275 }; 
```

This should accomplish the same effect.